be/src/exprs/vectorized/bitmap_functions.cpp

```diff
@@ -366,4 +367,37 @@ ColumnPtr BitmapFunctions::bitmap_min(FunctionContext* context, const starrocks:
     return builder.build(ColumnHelper::is_all_const(columns));
 }
 
+ColumnPtr BitmapFunctions::base64_to_bitmap(FunctionContext* context, const starrocks::vectorized::Columns& columns) {
+    ColumnViewer<TYPE_VARCHAR> viewer(columns[0]);
+    size_t size = columns[0]->size();
+    ColumnBuilder<TYPE_OBJECT> builder(size);
+    for (int row = 0; row < size; ++row) {
+        if (viewer.is_null(row)) {
+            builder.append_null();
+            continue;
+        }
+
+        auto src_value = viewer.value(row);
+        if (src_value.size == 0) {
+            builder.append_null();
+            continue;
+        }
+
+        int src_len = src_value.size;
+        std::unique_ptr<char[]> p;
+        p.reset(new char[src_len + 3]);
```

Comment:
why not use a `std::string` ?

Reply:
base64_decode2 need char*, we need string and convert?